modules/fluid_properties/doc/content/source/fluidproperties/SalineMoltenSaltFluidProperties.md

```diff
@@ -0,0 +1,42 @@
+# SalineMoltenSaltFluidProperties
+
+!syntax description /FluidProperties/SalineMoltenSaltFluidProperties
+
+The mole fractions of the various elements / molecules present in the salt must sum to one.
+The following properties can be computed:
+
+- density
+- isobaric specific heat
+- enthalpy
+- viscosity
+- thermal diffusivity
+
+!alert note
+Saline must be installed alongside MOOSE using the `contrib/saline` submodule in the fluid properties
+module to be able to use this fluid property object. To initialize/download the saline submodule:
+
+```bash
+git submodule update --init contrib/saline
+```
```

Comment:
The code block gets rendered outside of the alert block. To fix, change "!alert" to "!alert!" and then add a line with "!alert-end!" after your code block.